astropy/io/ascii/tests/test_read.py

```diff
@@ -1283,3 +1284,27 @@ def test_unsupported_read_with_encoding(tmpdir):
     with pytest.raises(ascii.ParameterError):
         ascii.read('t/simple3.txt', guess=False, fast_reader='force',
                    encoding='latin1', format='fast_csv')
+
+
+def test_read_pandas_csv(tmpdir):
+    pandas = pytest.importorskip('pandas')
+
+    df = pandas.DataFrame()
+    df['stuff'] = [x for x in range(10)]
+    no_index_fname = os.path.join(str(tmpdir), 'no_index.csv')
```

Comment:
can't you just use `tmpdir.join(...)` without `os`?

Reply:
Ah yes, I keep forgetting about that. I thought for awhile there we were trying to maintain compatibility with older versions of pytest, but I guess that's not important anymore?

Reply:
we already require >3.1, so no need to optimize on old pytests....